src/main/java/com/minecolonies/network/messages/BuildToolPlaceMessage.java

```diff
@@ -169,6 +169,18 @@ private static void handleHut(@NotNull World world, @NotNull EntityPlayer player
 
             if (building != null)
             {
+                if(building.getTileEntity()!= null)
+                {
+                    Colony colony=ColonyManager.getColony(world,buildPos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'colony' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)